examples/data-objects/prosemirror/src/fluidBridge.ts

```diff
@@ -533,6 +535,7 @@ function generateFragment(segments: ISegment[]) {
                         openTop = undefined;
                     }
                     // Create the new node, add it to the top's content, and push it on the stack
+                    // eslint-disable-next-line no-case-declarations
```

Comment:
Can we fix these easily with braces around the case body?  Or is this depending on the visibility of the declaration?

Reply:
Great question. I don't think braces fixes it but I'll try.

Reply:
Braces worked; great idea!